addons/udes_stock/tests/test_picking_batch.py

```diff
@@ -1087,49 +1087,110 @@ class TestBatchGetNextTask(common.BaseUDES):
     @classmethod
     def setUpClass(cls):
         super(TestBatchGetNextTask, cls).setUpClass()
-        cls.pack_2prods_info = [{'product': cls.apple, 'qty': 4},
-                                {'product': cls.banana, 'qty': 4}]
+        Package = cls.env['stock.quant.package']
+
+        cls.package_a = Package.get_package("1", create=True)
+        cls.package_b = Package.get_package("2", create=True)
+        cls.apple_quant = cls.create_quant(
+            cls.apple.id,
+            cls.test_location_01.id,
+            4,
```

Comment:
Nit: As `4` is the quantity of apple and banana, maybe put this into a `product_qty` variable? Then if you wanted to change it for whatever reason you'd only need to change it in one place instead of 4.